src/librustdoc/html/render/mod.rs

```diff
@@ -808,7 +808,7 @@ function handleThemeButtonsBlur(e) {{
 themePicker.onclick = switchThemeButtonState;
 themePicker.onblur = handleThemeButtonsBlur;
 {}.forEach(function(item) {{
-    var but = document.createElement('button');
+    var but = document.createElement(\"button\");
```

Comment:
```suggestion
    var but = document.createElement("button");
```

This is inside a raw string.